rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-installing-rosa.adoc

```diff
@@ -1,12 +1,12 @@
 :_content-type: ASSEMBLY
 [id="rosa-installing-cli"]
-= Installing the ROSA CLI
+= Installing the {product-title} (ROSA) CLI (`rosa`)
```

Comment:
Valid use, but be careful of the parentheses causing rendering issues

Reply:
Working fine.